evm_loader/program/src/neon.rs

```diff
@@ -0,0 +1,55 @@
+//! NEON ELF
+#![allow(clippy::use_self,clippy::nursery)]
+
+use const_format::formatcp;
+
+const NEON_CONST_PKG_VERSION: &str = env!("CARGO_PKG_VERSION");
+const NEON_CONST_REVISION: &str = env!("NEON_REVISION");
+use crate::account_data::ACCOUNT_SEED_VERSION;
+const NEON_CONST_SEED_VERSION: &str = formatcp!("{:?}", ACCOUNT_SEED_VERSION);
+use crate::account_data::ACCOUNT_MAX_SIZE;
+const NEON_CONST_ACCOUNT_MAX_SIZE: &str = formatcp!("{:?}", ACCOUNT_MAX_SIZE);
+
+
+const fn create_byte_array<const SZ: usize>(src: &[u8]) -> [u8; SZ] {
+    let mut array: [u8; SZ] = [0; SZ];
+    let mut i = 0;
+    while i < SZ {
+        array[i as usize] = src[i as usize];
+        i += 1;
+    }
+    array
+}
+
+
+const fn size_of_byte_array(array: &[u8]) -> usize {
+    array.len()
+}
+
+
+const SZ_NEON_PKG_VERSION : usize = size_of_byte_array(NEON_CONST_PKG_VERSION.as_bytes());
+/// NEON VERSION
+#[no_mangle]
+#[used]
+pub static NEON_PKG_VERSION: [u8; SZ_NEON_PKG_VERSION] = create_byte_array::<SZ_NEON_PKG_VERSION>(NEON_CONST_PKG_VERSION.as_bytes());
```

Comment:
Why do we need these lines to add one parameter? it seems too waste and complex for version purposes. I think we can use `&str` export for our parameters. Or we need to write a script, which should be used to add a new parameter.

Reply:
The main problem was to understand what size the value is if it is passed by reference.
And if you pass it as an array of a known value, then this problem is not.
We know the size and read it from the binary

Reply:
Fixed by the new macro neon_elf!